inspirehep/config.py

```diff
@@ -889,3 +894,5 @@ def _(x):
     'stat.OT': 'Other',
     'stat.TH': 'Other'
 }
+
+ASSETS_DEBUG = True
```

Comment:
That should have been removed by me. But I think for development it makes
sense since it reduces page loading times massively :)

On Fri, 15 Apr 2016 11:17 Javier Martin Montull, <notifications@github.com>
wrote:

> In inspirehep/config.py
> <https://github.com/inspirehep/inspire-next/pull/1048#discussion_r59847739>
> :
>
> > @@ -889,3 +894,5 @@ def _(x):
> >      'stat.OT': 'Other',
> >      'stat.TH': 'Other'
> >  }
> > +
> > +ASSETS_DEBUG = True
>
> Is this a leftover? We have been usually adding it in our instance folder
> local config, but maybe it makes sense to have it here?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/inspirehep/inspire-next/pull/1048/files/e7da8d6ae9bb6be7870efc0408acfce5b9d3d938#r59847739>
>